test/python/tensorflow/README.md

```diff
@@ -68,5 +69,5 @@ tf_unittest_runner is primarily used to run tensorflow python unit tests using n
    ```
  -  List of tests to run can be listed in a text file and pass the file name 
      to  argument `--run_tests_from_file` to run. 
-
+ -  To verify the Op placement on ngraph while running the tests set NGRAPH_TF_LOG_PLACEMENT=1 NGRAPH_TF_DISABLE_DEASSIGN_CLUSTER=1
```

Comment:
`NGRAPH_TF_DISABLE_DEASSIGN_CLUSTER=1` should be set always, else ops might not land up on ngraph due to reassignment of singleton clusters

Reply:
Not related to changes in this PR, but in the readme doc generally:

```
What can be tested

Python tests using Tensorflow with nGraph embedded
Python tests by patching Tensorflow to allow runnings tests using nGraph. tf_unittest_ngraph.patch
```

nGraph embedded -- >nGraph embedded in Tensorflow (when built using option 3) //link this https://github.com/NervanaSystems/ngraph-tf#linux-instructions

Similarly for 2nd line, lets specify option 1 and 2